notifications_utils/jinja_templates/letter_pdf/_main_css.jinja2

```diff
@@ -193,5 +194,24 @@
     word-wrap: break-word;
     border-radius: 1.05em;
   }
+  .qrcode,
+  .qrcode‑placeholder {
+      display: block;
+      width: {{ line_height_raw * 4.25 }}pt;
+      height: {{ line_height_raw * 4.25 }}pt;
```

Comment:
This should be
```suggestion
      width: {{ line_height_raw * 5.25 }}pt;
      height: {{ line_height_raw * 5.25 }}pt;
```
as per https://trello.com/c/cMj9CTop/408-finalise-size-and-spacing-of-qr-codes#comment-6493182d313cc01d3f3ee18f